plugin/pxr/maya/lib/usdMaya/exportTranslator.cpp

```diff
@@ -61,6 +61,13 @@ MStatus UsdMayaExportTranslator::writer(
             theOption.clear();
             optionList[i].split('=', theOption);
             if (theOption.length() != 2) {
+                // We allow an empty string to be passed to exportRoots. We must process it here.
```

Comment:
I assume that the code duplication with the ADSK export translator is O.K., as we don't expect the Pixar code to be long-lived.

Reply:
Yea. Something I discovered while writing this code. Not in scope for this PR and most likely will end up going the path you described above.